functions/Test-DbaDatabaseOwner.ps1

```diff
@@ -109,14 +109,13 @@ that TargetLogin must be a valid security principal that exists on the target se
 					Continue
 				}
 			}
-			
+			#use online/available dbs
+            $dbs = $server.Databases | Where-Object {$_.Status -eq 'Normal'}
```

Comment:
Ditto with the Copy-Database thing where it says if its skipped. Sorry, Mike asked me yesterday if we'd like to implement an internal get-cleandatabase sort of script and I excitedly said yes not thinking that it's better to see what is skipped and perhaps why.